.travis.yml

```diff
@@ -127,6 +127,8 @@ script:
   - python -m pytest test/dataframe.py
 
   - python -m pytest python/ray/rllib/test/test_catalog.py
+  - python -m pytest python/ray/rllib/test/test_filters.py
+  - python -m pytest python/ray/rllib/test/test_optimizers.py
```

Comment:
Also test_evaluators? (we should really auto-add tests...)

Reply:
test_evaluators has `tf` dependency, but somehow it hangs forever when put into jenkins tests...

Reply:
The jenkins tests aren't quite as good at propagating error messages, since they are run inside of docker.. but anyway, TF should be installed in those containers, right? Since we run other jenkins tests that use TF.

Reply:
Yeah - it runs in jenkins but somehow hangs, not sure why - maybe a combination of `unittest` and `threading`